iocore/eventsystem/P_IOBuffer.h

```diff
@@ -694,6 +695,7 @@ TS_INLINE char &IOBufferReader::operator[](int64_t i)
   }
 
   ink_release_assert(!"out of range");
+  return default_ret[0];
```

Comment:
The real problem is ICC doesn't realize the assert never returns.

Although, I don't understand why `static char zwoop = '0';` `return zwoop;` wouldn't work, rather than messing about with an array.